semaphoreci.sh

```diff
@@ -75,7 +75,7 @@ for PREREQ in ${GCC_PREREQS}; do
 done
 
 ## Create the build directory.
-# Build typically takes around 10 minutes, could this be cached across CI runs?
+# Build typically takes around 10 minutes with -j4, could this be cached across CI runs?
 mkdir ${SEMAPHORE_PROJECT_DIR}/build
```

Comment:
I think this should be fine as a starting point anyway, it's already just under twice as fast as before.